templates/index.html

```diff
@@ -0,0 +1,243 @@
+      <p class="w-full text-xl font-light text-center pb-5">
+        {{organizations.length}} <b>organizations</b> are <b>supporting Kivy</b>
+      </p>
+      <a v-bind:href="organization.profile" class="w-1/2 md:w-1/3 lg:w-1/5 p-2 xl:p-5"
+        v-for="organization in organizations.slice(0, 5)">
+        <div class="bg-[#e8e8e8] shadow-xl rounded-lg w-full p-4">
+          <div class="flex justify-center">
+            <img class="w-24 h-24" v-bind:src="organization.account.imageUrl" />
+          </div>
+          <div class="flex justify-center items-center h-14">
+            <p class="text-center text-lg font-bold">
+              {{ organization.account.name }}
+            </p>
+          </div>
+          <div class="flex justify-center h-10 items-center">
+            <p class="text-lg text-lime-600 uppercase">
+              <span v-if="organization.tier">{{organization.tier.name}}</span>
+            </p>
+          </div>
+          <div class="flex flex-col justify-center items-center">
+            <p class="text-2xl">
+              ${{organization.totalDonations.value}}
+            </p>
+            <p class="text-xs">
+              Amount contributed
+            </p>
+          </div>
+        </div>
+      </a>
+      <div class="w-full flex text-center justify-end">
+        <a href="https://opencollective.com/kivy"
+          class="text-sm md:text-base font-light md:pr-2 xl:pr-5 pt-3 pb-5 underline">
+          <b>See all the {{organizations.length}} organizations supporting Kivy on OpenCollective</b>
+        </a>
+      </div>
+
+      <p class="w-full text-xl font-light text-center pt-10 pb-5">
+        {{individuals.length}} <b>individuals</b> are <b>supporting Kivy</b>
+      </p>
+      <a v-bind:href="'https://www.opencollective.com/' + individual.account.slug"
+        v-bind:title="individual.account.name + ', Total contributed: ' + individual.totalDonations.value + ' ' + individual.totalDonations.currency"
+        v-for="individual in individuals">
+        <div class="bg-[#e8e8e8] w-8 h-8 rounded-full flex justify-center items-center">
+          <img class="h-full w-full object-cover rounded-full" v-bind:src="individual.account.imageUrl" />
+        </div>
+      </a>
+      <div class="w-full flex text-center justify-end">
+        <a href="https://opencollective.com/kivy"
+          class="text-sm md:text-base font-light md:pr-2 xl:pr-5 pt-3 pb-5 underline">
+          <b>See all the {{individuals.length}} individuals supporting Kivy on OpenCollective</b>
+        </a>
+      </div>
+    </div>
+  </div>
+
+  [[ include "components/support_kivy_banner.html" ]]
+  [[ include "components/need_help.html" ]]
+  [[ include "components/footer.html" ]]
+
+  <script src="https://unpkg.com/vue@3"></script>
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+  <script>
+    let opencollective_graphql_url = "https://api.opencollective.com/graphql/v2";
+    const graphql_query =
```

Comment:
Shouldn't we have this entire script section in its own external JS file?

Reply:
Moved to `content/static/js/opencollective_app.js`